orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/rollingpush/WaitForNewInstanceLaunchTask.groovy

```diff
@@ -41,11 +41,12 @@ class WaitForNewInstanceLaunchTask implements RetryableTask {
   @Override
   TaskResult execute(Stage stage) {
     StageData stageData = stage.mapTo(StageData)
-    def response = oortService.getServerGroup(stageData.application, stageData.account, stageData.cluster, stage.context.asgName as String, stage.context.region as String, stageData.providerType ?: 'aws' )
+    def response = oortService.getServerGroup(stageData.application, stageData.account, stageData.cluster, stage.context.asgName as String, stage.context.region as String, stageData.cloudProvider ?: 'aws' )
 
     Map serverGroup = objectMapper.readValue(response.body.in(), Map)
 
-    Set<String> currentIds = serverGroup.instances*.instanceId
+    def serverGroupInstances = serverGroup.instances
```

Comment:
Same comment as above wrt handling `id` vs `instanceId` elsewhere such that it's not a concern for orca.